content/doc/book/pipeline/syntax.adoc

```diff
@@ -248,6 +248,18 @@ need to contain its own `agent` directive. For example: `agent none`
 label:: Execute the Pipeline, or stage, on an agent available in the Jenkins
 environment with the provided label. For example: `agent { label 'my-defined-label' }`
 
+node:: Execute the Pipeline, or stage, on an agent available in the Jenkins
+environment with the provided label, with additional options. For example:
```

Comment:
is `label` required inside `node`?  Or will it work without that? 
Maybe note that this is not required if `label` is the only parameter?